models/PartnerBillingTierPerLoan.v1.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "PartnerBillingTierPerLoan",
+  "type": "object",
+  "description": "Structure de facturation du partenaire, par type de prêt et par intervalles de montants de prêts. Si aucun type de prêt n'est précisé, la structure s'appliquera à tous les types de prêts.",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "Type de prêt.",
+      "enum": [
+        "FREE_LOAN",
+        "BRIDGE_LOAN",
+        "PTZ_LOAN",
+        "BOSS_LOAN"
+      ]
+    },
+    "tiers": {
+      "type": "array",
+      "items": {
+        "$ref": "./PartnerBillingModel.v1.json"
+      }
+    },
+    "flat_fee": {
```

Comment:
Why do we need to include again flat_fee/rate in this structure?
Wouldn't tiers be enough? I have the feeling it might overcomplexify things?

Reply:
It's more complex, but I'm not sure a pure tiers models permits to covers all types of billings.
I can remove it, and we re-add it only in case a customer ask for it ?